lib/mfg.c

```diff
@@ -82,8 +82,11 @@
 #define SWITCHTEC_I2C_PORT_BITMASK		0x0f
 #define SWITCHTEC_I2C_ADDR_BITSHIFT		22
 #define SWITCHTEC_I2C_ADDR_BITMASK		0x7f
+#define SWITCHTEC_I2C_ADDR_BITMASK_GEN5	0xff
 #define SWITCHTEC_CMD_MAP_BITSHIFT		29
+#define SWITCHTEC_CMD_MAP_BITSHIFT_GEN5	30
 #define SWITCHTEC_CMD_MAP_BITMASK		0xfff
+#define SWITCHTEC_CMD_MAP_BITMASK_GEN5		0x3fff
```

Comment:
Some editors show this line as aligned, others don't. So it should be OK to ignore this 'mis-align issue'.